### PR TITLE
Fix styling to AR modal in seating context

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

Issue 105 from gsheet.

### 🗒️ Description

A new class is being added through [ETP#1648](https://github.com/the-events-calendar/event-tickets-plus/pull/1648)

Based on this class, we fix some styling issues of the AR modal. We continue to keep it same size as the seating selection modal but applying some amends on top of it.

### 🎥 Artifacts 
image
image
image

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [x] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

### CODE REVIEW

File: src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss [hunk 1]
```diff
@@ -31,10 +31,44 @@
 	}
 }
 
+.tec-tickets-seating__modal--ar {
+	.tribe-tickets__tickets-item {
+		border-top: 0;
+	}
+
+	@media screen and (min-width:1024px) {
```
Comment (rafsuntaskin) [inline src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss:39]: can we re-use some of our defined media query vars maybe?
`event-tickets/common/src/resources/postcss/utilities/variables/_media-queries.pcss`

File: src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss [hunk 2]
```diff
@@ -31,10 +31,44 @@
...
+		.tribe-tickets__attendee-tickets-container {
+			display: flex !important;
```
Comment (rafsuntaskin) [inline src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss:41]: can we avoid the `!important` by targeting more specific selector with parent?

**Comment by rafsuntaskin** [comment]
GHA tests are not running, we need trigger them by making some PHP comments or changes.

File: src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss (hunk 1 above)
Comment (dpanta94) [reply src/Tickets/Seating/app/frontend/ticketsBlock/style.pcss:39]: I wasn't able to use them -_-

**Comment by dpanta94** [comment]
Closing in favor of issue 171 from the gsheet. The modal should be reverted to its original styling as can be seen currently in production code.

**Pull request closed without merging**